server/wizard/steps/step_webhook_events.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow"
+)
+
+const (
+	stepNameWebhookEvents = "webhook_events"
+
+	stepTitleWebhookEvents = "Select webhook events"
+
+	stepDescriptionWebhookEvents = `- Click **Add events** and select the "End Meeting" event
+- Then click **Done**, **Save**, and **Continue**
+
+%s`
+)
```

Comment:
Can we verify that the webhook was successfully created by e.g. checking if an event was delivered?

Reply:
@hanzei I don't think zoom supports this. Do we want to include this as more of a manual process?

Reply:
Could we implement a hybrid approach where the user has to manually trigger a webhook event by e.g. running `/zoom start` and we then problematically verify that a webhook event was revived.

Reply:
@hanzei Something like:

> To test your connection with zoom, run  `/zoom start` to start a meeting, or click the Zoom icon in the channel header.

Then in the output, we include whether the webhook was successfully verified or not.